src/core.js

```diff
@@ -0,0 +1,38 @@
+// TODO: non-working code. Implement after accepting an API.
+
+var modules = {};
```

Comment:
For mutable variables, prefer `let` over `var`.

Reply:
Done.